ref/anbox-https-api.md

```diff
@@ -240,9 +240,9 @@ Return value for `curl -s --unix-socket /run/user/1000/anbox/sockets/api.unix -X
 
 After this, when opening a camera application, the uploaded image should be displayed in the preview.
 
-Here is a caveat about the size of a JPEG image to be uploaded to Anbox. Irrespective of the screen orientation is in landscape or portrait, the size of the uploaded JPEG image must match one of the resolutions you got from the response to the camera info request, Anbox will rotate the image automatically for you based on current screen orientation.
+If a static image already exists in Anbox Cloud, when you issue the above request next time, the image will be overridden.
```

Comment:
I'm not sure if I understand that sentence correctly.

Should I read it as: "Existing static images will be overriden by Anbox Cloud."?

Reply:
Maybe let's put it this way
```
If a static image already exists in Anbox container, when you issue the above request to upload a new image, the existing image will be overwritten.
```